tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -455,6 +455,158 @@ public function test_template_redirect() {
 		$this->assertStringContainsString( '(self.SWG_BASIC=self.SWG_BASIC||[]).push(basicSubscriptions=>{basicSubscriptions.init({"type":"NewsArticle","isPartOfType":["Product"],"isPartOfProductId":"' . $publication_id . ':openaccess","clientOptions":{"theme":"light","lang":"en-US"}});});', $footer_html );
 	}
 
+	public function data_product_ids__singular() {
+		return array(
+			'with no product ID configured'            => array(
+				array(),
+				'',
+				'openaccess',
+			),
+			'with product ID set in settings'          => array(
+				array(
+					'productID' => 'ABDCEFGH:advanced',
+				),
+				'',
+				'advanced',
+			),
+			'with product ID set in settings and post' => array(
+				array(
+					'productID' => 'ABDCEFGH:advanced',
+				),
+				'ABDCEFGH:openaccess',
```

Comment:
If the post product ID is set to `openaccess`, it won't have the publication ID prefix. So, I'd suggest either removing `ABDCEFGH:`, or changing `openaccess` to another value.

Reply:
You're absolutely right, updated, thank you!